src/app/scenes/block/block.component.html

```diff
@@ -61,10 +57,21 @@ <h1>Explore your block</h1>
           </ul>
         </li>
         <li>
-          <p>Extra Data</p>
-          <b>{{block.extra_data }}</b>
-        </li>
-      </ul>
+        <ul>
+          <li>
+            <p>Extra Data</p>
+            <b>{{block.extra_vanity }}</b>
```

Comment:
Can we use `extra_vanity` on the home page block table as well? And really anywhere else `extra_data` is being used. I'm not sure there is any use for it on the front end in the raw form.